### PR TITLE
Test cases + testing script

### PR DESCRIPTION
**Pull request by t-a-ma**
- added succ and fail test cases
- added additional tests inputs and outputs for evaluator 
- tests and fail tests added into  makefile 
- testscript now can pull succ and fail test names

### CODE REVIEW

File: test_cases/fail-cons1.bs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+/*
+    use cons on non-list variable
+*/
+
+int a;
+a = 3;
+5 :: a;
```
Comment (alanluc) [inline test_cases/fail-cons1.bs]: think this test should pass, we are not yet checking for whether a is a list

File: test_cases/fail-int1.bs [hunk 2]
```diff
@@ -0,0 +1,5 @@
+/*
+    type difference between variable and value
+*/
+int fail;
+fail  = "not int";
```
Comment (alanluc) [inline test_cases/fail-int1.bs]: same thing here, we don't care about type checking yet

File: test_cases/test-assignfunction1.bs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+/*
+    Creates a function that adds two 
+*/
+
+int add(int a, int b) {
+    return a + b;
+}
+
+
+int addSeven (function add, int c) {
```
Comment (alanluc) [inline test_cases/test-assignfunction1.bs:10]: shit we need to flesh out the syntax for creating a function that takes a function as a formal parameter

File: test_cases/test-lists1.bs [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/*
+    Creates two variables and a function that returns void.
+    Then creates a list consisting of the two variables, a string, and a function.
+*/
+
+
+int four = 4;
+int hello = "hello"
+void helloWorld() {
+    print(helloWorld)
```
Comment (alanluc) [inline test_cases/test-lists1.bs]: semicolon

File: test_cases/test-lists1.bs [hunk 5]
```diff
@@ -0,0 +1,14 @@
+/*
+    Creates two variables and a function that returns void.
+    Then creates a list consisting of the two variables, a string, and a function.
+*/
+
+
+int four = 4;
+int hello = "hello"
```
Comment (alanluc) [inline test_cases/test-lists1.bs]: semicolon

File: test_cases/test-assignfunction1.bs (hunk 3 above)
Comment (t-a-ma) [reply test_cases/test-assignfunction1.bs:10]: so remove for now?

Comment (alanluc) [reply test_cases/test-assignfunction1.bs:10]: not sure, but we need to flesh it out

Comment (t-a-ma) [reply test_cases/test-assignfunction1.bs:10]: can write another test for this submission and flesh this out later

Comment (alanluc) [reply test_cases/test-assignfunction1.bs:10]: ok

**Pull request merged**